carta/session.py

```diff
@@ -383,6 +383,19 @@ def append_image(self, path, hdu=""):
         """
         return Image.new(self, path, hdu, True)
 
+    def open_images(self, *paths):
+        """Open multiple images.
+
+        Parameters
+        ----------
+        paths : {0}
+            The paths to the image files, either relative to the session's current directory or an absolute path relative to the CARTA backend's root directory. Separate them with comma.
+        """
+        path_list = list(paths)
```

Comment:
If this were a comma-separated string, this would not do the right thing -- it would split the string into a list of individual characters. But this is in fact a tuple of strings, so this line is unnecessary -- you can do everything with a tuple below that you can do with a list, so the cast is not required. You'd only have to do that if you needed to modify the list in-place.

Use `paths` directly below.